client/rest-high-level/src/main/java/org/elasticsearch/client/LicenseClient.java

```diff
@@ -98,6 +101,28 @@ public void getLicenseAsync(GetLicenseRequest request, RequestOptions options, A
             response -> new GetLicenseResponse(convertResponseToJson(response)), listener, emptySet());
     }
 
+    /**
+     * Enables and starts a trial license for the cluster.
+     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
+     * @return the response
+     * @throws IOException in case there is a problem sending the request or parsing back the response
+     */
+    public PostStartTrialResponse postStartTrial(PostStartTrialRequest request, RequestOptions options) throws IOException {
+        return restHighLevelClient.performRequestAndParseEntity(request, RequestConverters::postStartTrial, options,
+            PostStartTrialResponse::fromXContent, singleton(403)
```

Comment:
If the trial is already started it will return a 403 code but without the exception structure that hlrc is expecting